.github/workflows/security-scan.yml

```
@@ -0,0 +1,62 @@
+            // Check for critical or high severity
+            const criticalVulns = vulnerabilities.vulnerabilities.filter(v => 
+              v.severity === 'critical' || v.severity === 'high'
+            );
+
+            if (criticalVulns.length > 0) {
+              console.log('Found critical/high vulnerabilities!');
+              
+              // Create GitHub issue
+              execSync(`gh issue create --title "🔒 Security: Critical vulnerabilities found" --body "Security scan found ${criticalVulns.length} critical/high severity vulnerabilities.\\n\\nPlease review and fix before merging.\\n\\nRun \`pnpm audit\` locally for details." --label security --assignee @irfandi`, { stdio: 'inherit' });
+            }
+        github-token: ${{ secrets.GITHUB_TOKEN }}
```

actionlint (1.7.10)

unexpected key "github-token" for step to execute action. expected one of "continue-on-error", "env", "id", "if", "name", "timeout-minutes", "uses", "with"

(syntax-check)